integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/standardpartition/StandardPartitionTableQueryTestCase.scala

```diff
@@ -256,9 +256,11 @@ test("Creation of partition table should fail if the colname in table schema and
       """.stripMargin)
     sql("insert into partitionTable select 1,'huawei','abc'")
     checkAnswer(sql("show partitions partitionTable"), Seq(Row("email=abc")))
-    intercept[Exception]{
+    val e = intercept[AnalysisException]{
```

Comment:
@xubo245 
sql("alter table partitionTable PARTITION (email='abc') rename to PARTITION (email='def)")

closing single quote is missing after sql("alter table partitionTable PARTITION (email='def) rename to PARTITION (email='def)") it is going to pass anyhow